### DISCUSSION TITLE
[Art competition] Cryptography

### DISCUSSION THREAD

**Discussion by bongjunj** [prosyslab-classroom/cs348-information-security#406] [Art competition]
![ai-art-cryptography-scale](https://github.com/prosyslab-classroom/cs348-information-security/assets/7106939/0ab4a81e-6334-4228-bd4c-e2e5a71bdc21)

## Short Description
For me, cryptography means [A robot and a lot of puzzle blocks on each side weighing scales, respectively, American cartoon style. The scales are tilted towards the side with the puzzle blocks. The background is filled with mathematical symbols and equations.]".

## Long Description (up to 3 sentences)

To design a cryptography system, we make a mathematical puzzle which is very hard for a computer to solve efficiently. But as a computer gets faster, the system can be broken (e.g. MD-5, SHA-1). This is like weighing two different things on each side of a weighing scales (difficulty vs computer performance).

**Comment by KihongHeo** [comment]
Nice. It seems that this poor DALL-E does not understand what "mathematical symbols and equations" mean.
